policies/testing.md

```diff
@@ -42,6 +42,46 @@ A test may be omitted where writing the test would result in disproportionately
 more effort than writing the code being tested. For example, difficult to
 reproduce error conditions.
 
+Triage Labels
+-------------
+
+Before approving a PR, the applicability of the testing policy to the PR must
+be assessed and an appropriate label applied to the PR. One of the following
+labels must be applied:
+
+- `tests: present`, to be added where suitable tests are included in the same PR.
+- `hold: tests needed`, where the testing policy requires tests but they are not
+  currently included in a PR.
+- `tests: exempted`, where tests are not included in a PR but are not required
+  by this policy (for example, because the PR concerns only documentation,
+  or because writing the test would result in disproportionately more effort;
+  see above).
+- `tests: deferred`, where the tests are required by the testing policy
+  but it is intended to add these tests in a subsequent PR. The label should
+  be removed (for example from a closed PR) once subsequent tests have
+  been merged.
```

Comment:
How do we deal with situations where the PR author considers that the current test suite is sufficient?

Reply:
I think this would come under `tests: present`.

Reply:
Could it fail to merge without one of these labels being set, with a printed error?

Reply:
How would you want to do this. It would require the GHE hooks to look at the PRs in the merged commits which I do not think we want.

Reply:
> I think this would come under tests: present.

It may also be "tests: exempted". For example a refactor is specifically exempted.

Reply:
I agree with @t8m re automatic checks, because of all the churn (and because github has a 5s time limit on pre-receive hooks, where those checks need to be placed)